src/Microsoft.AspNetCore.Razor.Tools/GenerateCommand.cs

```diff
@@ -155,6 +158,11 @@ protected override bool ValidateArguments()
             var engine = RazorProjectEngine.Create(configuration, compositeFileSystem, b =>
             {
                 b.Features.Add(new StaticTagHelperFeature() { TagHelpers = tagHelpers, });
+
+                if (GenerateDeclaration.HasValue())
+                {
+                    b.Features.Add(new EliminateMethodBodyPass());
+                }
```

Comment:
I see now, yeah I'd really prefer that this be part of `RazorCodeGenerationOptions`. If that's not feasible then log a follow up. Putting this feature in the CLI tool means that we can't really use it in tests or in other tools (like Ajay's thing).